app/views/claims/student_loan_amount.html.erb

```diff
@@ -13,11 +13,9 @@
           </h1>
 
           <span class="govuk-hint">
-            If you do not know the exact amount, check your P60, your annual
-            student loan statement, or contact the
-            <a href="https://www.slc.co.uk/students-and-customers/contact-information-for-customers/loan-repayment-enquiries.aspx" class="govuk-link">
-              student loans company
-            </a>.
+            Check your annual student loan statement, or your P60 if you had only one employer. You can also add up the
```

Comment:
Just wondering if this might be easier to read if it was broken into two paragraphs? It's quite a chunky block of text as it stands

Reply:
Ok, I'll double check with Faye 👍 

This is what it will look like:

<img width="1785" alt="Screenshot 2019-10-23 at 16 36 40" src="https://user-images.githubusercontent.com/2804149/67410005-54abb100-f5b3-11e9-8c38-2b404af02d79.png">

Reply:
I think that seems like a winner

Reply:
Sweet 👍

Reply:
Faye gave it the thumbs up, updated the commit.

As an aside I noticed we're not using the `aria-describedby` attribute on our inputs so I created a card for it: https://trello.com/c/4vGvgVmZ/882-add-aria-describedby-to-input-elements-with-hints